corral/test/integration/test_info.pony

```diff
@@ -0,0 +1,35 @@
+use "files"
+use "ponytest"
+use ".."
+use "../../util"
+
+class TestInfo is UnitTest
+  fun name(): String => "integration/info"
+  fun apply(h: TestHelper) =>
+    h.long_test(2_000_000_000)
+    Execute(h, recover [
+      "info"
+      "--bundle_dir"; Path.join(TestDir.path, "empty-deps")
+    ] end, CheckInfo)
+
+class CheckInfo is Checker
+  fun tag apply(h: TestHelper, ar: ActionResult) =>
+    h.assert_eq[I32](0, ar.exit_code)
+    h.assert_true(ar.stdout.contains("info: {}"))
+    h.complete(ar.exit_code == 0)
+    //h.env.out.print(ar.stdout)
```

Comment:
there's a lot of these prints commented out. 

what purpose do they serve? i'd like to not see commented out code.

Either we don't need it or we do.

Reply:
Not really needed. I just used them to see the output as I was writing the asserts. I'll remove them.